documentation/source/getting_started/quick_install.rst

```diff
@@ -0,0 +1,23 @@
+You can install OpenEBS instantly using the following procedure.
+
+Prerequisite:
+-------------
+You must have Kubernetes installed.
+
+1. Install Open-ISCSI using the following commands at the command prompt:
+
+   **On Ubuntu -**
+
+   * sudo apt-get update 
+   * sudo apt-get install open-iscsi
+
+   **On CoreOS - ??**
```

Comment:
remove this line. I will put in an issue(s) to include the steps for other operating systems.